src/Model/AuthToken.php

```diff
@@ -2,7 +2,20 @@
 
 namespace KolayIK\Auth\Model;
 
-class AuthToken extends \Illuminate\Database\Eloquent\Model
+use Illuminate\Database\Eloquent\Builder;
+use Illuminate\Database\Eloquent\Model;
+
+/**
+ * @property mixed|string token
+ * @property mixed|string user_id
+ * @property Carbon|mixed expiration_date
+ * @property mixed|string ip_address
+ * @property Carbon|mixed created_at
+ * @property Carbon|mixed updated_at
```

Comment:
Why there is mixed type for all of them? If they are nullable then you should add `type|null` why mixed is there?